script/.dockerignore

```diff
@@ -0,0 +1 @@
+libs/
```

Comment:
somewhat confused by this directory being called 'script'

Reply:
would `bin` be better?